masterlist.yaml

```
@@ -508,6 +478,11 @@ common:
       - lang: zh_CN
         text: '需要一个补丁来使这个mod与%1%完全兼容。目前没有可供下载的。'
 
+  - &patchUnavailable_Requiem
```

Just to keep the masterlists' style consistent, derivative anchors immediately follow their source anchor or other derivatives. Also, anchor names are camel case unless two uppercase letters from two separate words are joined (e.g., `requiresMCM_X`). Not a blocking change though.

```suggestion
  - &patchUnavailableRequiem
```